docs/achtergronddocumentatie/artefacten/0138.md

```diff
@@ -1,25 +1,43 @@
 ---
 layout: page-with-side-nav
 title: F0138 - Vraag partij op
-date: 28-03-2022
+date: 07-12-2023
 ---
 
 # F0138 - Vraag partij op
 
 ## Functionaliteit
 
-De functie retourneert de gevraagde gegevens van de `partij`.
+De functie retourneert de gevraagde gegevens van de `partij` op basis van de ID.
```

Comment:
Ik weet niet of ik de conventies helemaal doorgrond; zou `id` hier geen kleine letters moeten moeten hebben?

Reply:
Aangepast